Vokal-Swift.xctemplate/MainAPIUtility.swift

```diff
@@ -171,10 +187,10 @@ class MainAPIUtility {
         }
     }
     
-    func getJSON(from path: String,
+    func getJSON<T>(from path: String,
                  headers: [HTTPHeaderKey: HTTPHeaderValue],
```

Comment:
Does this need to be reindented - would apply to some other methods in this file